lib/ae_mdw/aex141.ex

```diff
@@ -127,12 +127,14 @@ defmodule AeMdw.Aex141 do
         {:ok, m_limits} -> m_limits
       end
 
-    %{
-      token_limit: token_limit,
-      template_limit: template_limit,
-      limit_txi: txi,
-      limit_log_idx: log_idx
-    }
+    if token_limit != nil or template_limit != nil do
```

Comment:
@marc0olo this is the condition proposed to show the map, otherwise it shows `limits: nil`

Reply:
I was just thinking about explicitness vs. implicitness when it comes to differentiation between the `mintable` extension and the `mintable_templates` extension.

from my perspective there is always `token_limit` **OR** `template_limit` as those limit types cannot/should not be mixed among these extensions.

for me this is generally fine. if we'd want to be more explicit we'd add extension related attributes in their own section instead of delivering these values in a generic way without relation to any extension. that is what I was thinking of. but I don't have a strong opinion.

Reply:
it's fine, thanks. it's only matter of indexing using the same record to keep one limit or the other.

Reply:
agree, doesn't make sense to split the limit info into different sections. too little differentiation. domain knowledge is required anyway 😅